ThreadSafe/Map/Map.go

```diff
@@ -32,6 +32,35 @@ func New(initCap int) *ThreadSafeMap {
 	return &ThreadSafeMap{items: items}
 }
 
+// Wraps the map as ThreadSafe.
+// > You should not use that map as general one anymore. Or just use MakeThreadSafeCopy
+func MakeItThreadSafe(m map[string]interface{}) *ThreadSafeMap {
```

Comment:
What will be made threadsafe? Wouldn't something like MakeMapThreadSafe be more clear?